packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/location-listing.twig

```diff
@@ -9,7 +9,9 @@
   {% endif %}
   <div class="ma__location-listing__columns">
     <div class="ma__location-listing__container">
-      <div class="ma__location-listing__map">
+      <div class="ma__location-listing__map" aria-label="{{leafletMap.title}}">
```

Comment:
not seeing this label being rendered https://mayflower.digital.mass.gov/patternlab/b/feature/DP-22210_fix-focus-issue/?p=pages-location-listing